.rubocop.yml

```diff
@@ -11,6 +11,8 @@ Layout/FirstArrayElementIndentation:
   EnforcedStyle: consistent
 Layout/LineLength:
   Max: 100
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
```

Comment:
this cop is only relevant to a change that I made temporarily, but i'm going to leave this specification in here, anyway